riot/riot.ml

```diff
@@ -27,6 +33,7 @@ let run ?(rnd = Random.State.make_self_init ()) ?workers main =
   Scheduler.Pool.set_pool pool;
 
   let _pid = _spawn ~pool ~scheduler:sch0 main in
+  let () = started := true in
```

Comment:
**small fix.** I think we can move this line up to line 18 (so right after the if) and use the syntax `started := true;`